ddspipe_core/test/unittest/efficiency/MapPayloadPoolTest.cpp

```diff
@@ -51,15 +51,15 @@ class MockMapPayloadPool : public MapPayloadPool
     }
 
     uint64_t reference_count(
-            const Payload& payload)
+            const eprosima::fastdds::rtps::SerializedPayload_t& payload)
```

Comment:
You can delete all references to `eprosima` namespace:

```suggestion
            const fastdds::rtps::SerializedPayload_t& payload)
```